__init__.py

```diff
@@ -10,6 +10,8 @@
 
 from dotenv import load_dotenv
 
+from version import __version__
```

Comment:
This would now be `v0.0.0` now when there is no git tag, right? Is that expected?